enforcer/lookup/lookup.go

```diff
@@ -139,23 +140,35 @@ func (m *PolicyDB) AddPolicy(selector policy.TagSelector) (policyID int) {
 
 }
 
+func (m *PolicyDB) keyValueFromString(tag string) (key, value string) {
+
+	parts := strings.SplitN(tag, "=", 2)
```

Comment:
We actually allow that. The first equal is the one that matters. We do allocate a=b=c .. then a is the key and b=c is the value